.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
   "env": {
+    "es6": true,
```

Comment:
that enable es6 syntax to pass eslint rules (for example es6 classes, lambdas). eslint is only done on our quicktests, and since our quicktests use modern browsers in sauce and locally, this should be fine. Welcome to 2017!